vocabularies/mining-survey-status.ttl

```diff
@@ -0,0 +1,63 @@
+﻿@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> ..
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix sstatus: <http://linked.data.gov.au/def/mining-survey-status/> 
+
+<http://linked.data.gov.au/def/mining-survey-status> a owl:Ontology .
+
+sstatus:conceptScheme a skos:ConceptScheme;
+  dct:created "2019-04-09T03:31:01.023Z"^^xsd:dateTime;
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  skos:definition "The stage in the lifecycle of an exploration survey or activity (including research activities)"@en;
+  skos:prefLabel "Mining Survey or Activity Status"@en .
+
+sstatus:proposed a skos:Concept;
+  skos:definition "A survey or activity has been proposed, but has not yet commenced."@en;
+  skos:notation "PRO"@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Proposed"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:current-in-progress a skos:Concept;
+  skos:notation "INPRO"@en;
+  skos:definition "A survey or activity is currently in progress."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "In-progress"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:current-on-hold a skos:Concept;
+  skos:notation "ONHOL"@en;
+  skos:definition "A survey or activity is current, but on-hold."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "On-hold"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:interim-report-received a skos:Concept;
+  skos:notation "INRPT"@en;
+  skos:definition "An interim report for the survey or activity has been received."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Interim report received"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:data-outstanding a skos:Concept;
+  skos:notation "OS"@en;
+  skos:definition "The survey or activity has been completed, but data or a final report is outstanding."@en;
+  skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Data outstanding"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:completed a skos:Concept;
+  skos:notation "COMP"@en;
+  skos:definition "The survey or activity has been completed, and a final report has been received."@en;
+    skos:inScheme sstatus:conceptScheme;
+  skos:prefLabel "Completed"@en;
+  skos:topConceptOf sstatus:conceptScheme .
+
+sstatus:abandoned a skos:Concept;
+  skos:notation "ABAND"@en;
+  skos:definition "A survey or activity has been abandoned, because, although commenced, it was not completed due to unforeseen issues or for other reasons."@en;
```

Comment:
Should "although commenced" be removed? e.g. a company lodged a Notice of Intention to Drill and then never drilled.

Reply:
changed